modules/installation-osp-deploying-provider-networks-installer.adoc

```diff
@@ -33,15 +33,19 @@ The `platform.openstack.apiVIP` and `platform.openstack.ingressVIP` properties m
         ...
         platform:
           openstack:
-            apiVIP: 192.0.2.13
-            ingressVIP: 192.0.2.23
+            apiVIPs: <1>
+              - 192.0.2.13 
+            ingressVIPs: <1>
+              - 192.0.2.23 
             machinesSubnet: fa806b2f-ac49-4bce-b9db-124bc64209bf
             # ...
         networking:
           machineNetwork:
           - cidr: 192.0.2.0/24
 ----
 
+<1> From {product-title} 4.12 or later, the `apiVIP` and `IngressVIP` configuration settings are deprecated. Instead, use a list format to enter values in the `apiVIPs` and `IngressVIPs` configuration settings.
```

Comment:
Some of these notes have a capitalization issue with IngressVIP(s). It should be a lowercase i in ingress.